README.md

```diff
@@ -58,3 +94,33 @@ The example with default language model and datasets.
 python evaluate_sequence_tagging.py
 ```
 
+# Passing task model and task training parameters to predefiend pipelines
+
+Model and training parameters can be controlled via `task_model_kwargs` and 
+`task_train_kwargs` parameters. 
+
+## Example with `polemo2` dataset.   
+
+```python
+from embeddings.pipeline.hugging_face_classification import HuggingFaceClassificationPipeline
+
+pipeline = HuggingFaceClassificationPipeline(
+    dataset_name="clarin-pl/polemo2-official",
+    embedding_name="allegro/herbert-base-cased",
+    input_column_name="text",
+    target_column_name="target",
+    output_path=".",
+    task_model_kwargs={
```

Comment:
You read my mind, I ran the example and then thought it would be great to pass params to the model, scrolled a couple of file down and magic, it is already done. Great job!